extensions/wikia/AdEngine/js/provider/gpt/googleSlots.js

```diff
@@ -0,0 +1,25 @@
+define('ext.wikia.adEngine.provider.gpt.googleSlots', function() {
```

Comment:
Do you think it makes sense to use here also `wikia.log` module and log some simple messages useful for debugging?